pymc_marketing/clv/models/pareto_nbd.py

```diff
@@ -219,31 +281,117 @@ def build_model(  # type: ignore
         with pm.Model(coords=self.coords) as self.model:
             # purchase rate priors
             r = self.model.register_rv(self.r_prior, name="r")
-            alpha = self.model.register_rv(self.alpha_prior, name="alpha")
+            if self.pr_covar_columns is not None:
+                alpha0 = self.model.register_rv(self.alpha0_prior, name="alpha0")
```

Comment:
instead of `alpha0` I would call this `alpha_scale`. The zero doesn't convey any meaning. Same for `beta0`